packages/eslint-config-nimble-react/lib/rules/prettier.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+module.exports = {
+  plugins: ['prettier'],
+  extends: ['plugin:prettier/recommended'],
+  rules: {
+    'prettier/prettier': ['error', { singleQuote: true }],
```

Comment:
Should we also add rules for semicolons and tailing commas?

```suggestion
    'prettier/prettier': ['error', { singleQuote: true, semi: true, trailingComma: 'es5' }],
```

Reply:
Added in https://github.com/nimblehq/eslint-config-nimble/pull/32/commits/e56eff6eef20fe9f3afcef5c42b4a08426377366